### PR TITLE
Removed dependency on DateToolsSwift

### PR DESCRIPTION
**Pull request by grosch**
Not sure how to test this, but I removed the DateToolsSwift package and just used the standard methods Foundation provides.  Most of the stuff that library was doing is pretty simple in the newer versions of Swift, so I think it's good to remove that dependency.  

Happy to do more work on this.

### CODE REVIEW

**Comment by grosch** [comment]
Relates to issue #285

**Comment by richardtop** [comment]
Hi, thanks a lot for this pull request. Removing DateTools has been definitely in the consideration.
Overall it looks pretty good. Could you please add the following to your pull request:

1. Remove DateTools from the `.podspec` file, as to not to include this framework when installing with the CocoaPods
2. Rewrite sample apps to not use the DateTools. Currently, your pull request doesn't compile with the sample apps (because sample apps use `DateTools`).
3. To simplify the part 2 (rewrite of the apps), feel free to leave just the `CustomCalendarExampleController` and remove the `ExampleController` and the `ExampleNotificationController`.

Really good job and I'm looking forward to merging it as soon as those issues are fixed.

**Comment by grosch** [comment]
There ya go.

File: CalendarKitDemo/CalendarApp/CustomCalendarExampleController.swift [hunk 1]
```diff
@@ -46,73 +45,29 @@ class CustomCalendarExampleController: DayViewController, DatePickerControllerDe
...
+    fmt.dateStyle = .none
+    fmt.timeStyle = .short
+
+    return fmt
   }()
-  
+
   override func loadView() {
-    calendar = customCalendar
+    calendar.timeZone = TimeZone(identifier: "Europe/Paris")!
```
Comment (richardtop) [inline CalendarKitDemo/CalendarApp/CustomCalendarExampleController.swift:57]: Could you please bring back the custom calendar implementation, as it shows how to launch CK and change something else but timezone. E.g. when a completely custom calendar is needed.

File: CalendarKitDemo/CalendarApp/CustomCalendarExampleController.swift [hunk 2]
```diff
@@ -124,24 +79,22 @@ class CustomCalendarExampleController: DayViewController, DatePickerControllerDe
   }
   
   private func generateEventsForDate(_ date: Date) -> [EventDescriptor] {
-    var workingDate = date.add(TimeChunk.dateComponents(hours: Int(arc4random_uniform(10) + 5)))
+    var workingDate = Calendar.current.date(byAdding: .hour, value: Int.random(in: 1...15), to: date)!
```
Comment (richardtop) [inline CalendarKitDemo/CalendarApp/CustomCalendarExampleController.swift:82]: Also, please use your custom calendar here, not the `current` one

**Pull request merged**